cordova/config.xml

```diff
@@ -63,16 +63,20 @@
     </feature>
     <preference name="Orientation" value="portrait" />
     <preference name="Fullscreen" value="true" />
-    <config-file file="*-Info.plist" mode="merge" target="NSCameraUsageDescription">
+    <preference name="CAMERA_USAGE_DESCRIPTION" default=" " />
```

Comment:
Do we need to put text inside the descriptions?

Reply:
No.. the description text is inside the `<string>` tag.